addons/web/static/src/views/fields/many2one_avatar/many2one_avatar_field.xml

```diff
@@ -20,8 +20,13 @@
         </xpath>
         <xpath expr="//span[hasclass('o_m2o_avatar_empty')]" position="replace">
             <t t-if="props.record.data[props.name] === false and props.isEditable">
-                <a t-on-click.stop.prevent="openPopover" tabIndex="-1" href="#" title="Assign"
-                   aria-label="Assign" class="fa fa-user-plus o_quick_assign d-flex align-items-center"/>
+                <a t-on-click.stop.prevent="openPopover"
```

Comment:
Let's try to avoid attributes' indentation changes unless strictly necessary.
I do agree that's more readable, but it makes reading the diff way harder.